llm/llm.go

```diff
@@ -50,92 +50,88 @@ func New(model string, adapters, projectors []string, opts api.Options) (LLM, er
 	}
 
 	vram, _ := gpu.CheckVRAM()
-	size := ggml.Size
+	info := gpu.GetGPUInfo()
+
+	mem := int64(396 * 1024 * 1024)
```

Comment:
Where did 396 MiB come from? Is it really safe to assume this for all projectors or just the ones we have so far?

Reply:
This is the overhead of initializing the GPU and unrelated to projectors. This will likely need to be at least GPU vendor specific but the only reliable number I can find is for NVIDIA

Reply:
Is this once per model load, or is it once per GPU?

Maybe add a comment?